Dockerfile

```diff
@@ -27,8 +27,19 @@ RUN apt-get update -y && \
     rm -rf /var/lib/apt/lists/*
 
 # Serve frontend application
-FROM docker.io/library/nginx:1.19
+FROM docker.io/library/nginx:1.25
 COPY --from=react-build /code/reana-ui/build /usr/share/nginx/html
 COPY nginx/reana-ui.conf /etc/nginx/conf.d/default.conf
 EXPOSE 80
 CMD ["nginx", "-g", "daemon off;"]
+
+LABEL org.opencontainers.image.authors="team@reanahub.io"
+LABEL org.opencontainers.image.created="2023-12-12"
+LABEL org.opencontainers.image.description="REANA UI"
```

Comment:
What about:

- "REANA reproducible analysis platform - web user interface component"

We could use the same description naming technique elsewhere, e.g. ...job controller component, ...Snakemake workflow engine component, etc.